### ISSUE TITLE
0.17.4 Invalid Path

### ISSUE THREAD

**Issue by polonel** [apidoc/apidoc#590]
After Updating to version 0.17.4 I started getting this error:
error: ENOENT: no such file or directory, open 'C:\NodeApps\trudesk\src\controllers\api\C:\NodeApps\trudesk\src\controllers\api\v1\groups.js'

Command:
``` 
apidoc -i src/controllers/api -d apidocs/
error: ENOENT: no such file or directory, open C:\NodeApps\trudesk\src\controllers\api\C:\NodeApps\trudesk\src\controllers\api\v1\groups.js'

```

Its putting the "src" path before filepath, thus making it fail to find the file.

**Comment by t33m** [comment]
Yep, got this error after installing on mac.

**Comment by phips28** [comment]
same for me

**Comment by rottmann** [comment]
Fixed in 0.17.5 (new klaw-sync 1.1 did now normalize rel and absolute dirs).

**Comment by polonel** [comment]
Confirmed this has been fixed. Closing Issue.
